### PR TITLE
FRON-48 GDPR approval 

### PR DESCRIPTION
**Pull request by MaliniakovaKarin**

### CODE REVIEW

**Review by MCFreddie777 — DISMISSED** [review]

File: src/views/Upload.vue [hunk 1]
```diff
@@ -199,33 +199,70 @@ export default {
...
+							využitá.
+						</p>
+						<h2 class="text-xl font-bold">Spracovanie osobných údajov</h2>
+						<p class="mt-3">
+							Súhlasom so spracovaním osobných údajov udeľuje dotknutá
+							osoba svoj výslovný súhlas s vyššie uvedeným spracovaním.
+							Súhlas možno vziať kedykoľvek späť, napísaním emailu s
+							predmetom správy „Odvolanie súhlasu so spracovaním osobného
+							údaju“ na emailovú adresu <a href="mailto:tim10fiit@googlegroups.com" class="hover:underline text-primary-500">tim10fiit@googlegroups.com</a>.
+						</p>
```
Comment (Hemii) [inline src/views/Upload.vue:221]: Should we have these terms in English as well, if English is chosen ?

File: src/i18n/languages/english.js [hunk 2]
```diff
@@ -31,7 +31,10 @@ const upload = {
 	dragFilesHere: 'Drag files here',
 	chooseFiles: 'Choose files',
 	uploadTextDisabledMessage: 'Some files were uploaded. You might either upload file or paste text.',
-	uploadTextPlaceholder: 'Paste text here..'
+	uploadTextPlaceholder: 'Paste text here..',
+	emailPolicyTitle: '<h1 class="text-2xl font-bold">Conditions for the processing of personal data</h1>',
```
Comment (MCFreddie777) [inline src/i18n/languages/english.js]: Please, do not add the html code to the translations. What if we wanted to changer the html tag or css classes? We would have to make it in both places.. What if we had thousands of language translations?

File: src/views/Upload.vue [hunk 3]
```diff
@@ -199,33 +199,47 @@ export default {
...
-			this.$axios
-				.post(
-					'/api/submissions/',
-					isFileUpload ? formData : requestBody,
-					{
-						headers
+			this.$store.dispatch('ModalStore/setModal', {
+				type: 'html',
+				htmlTemplate: {
+					header: this.$i18n.t('emailPolicyTitle'),
```
Comment (MCFreddie777) [inline src/views/Upload.vue]: Please, use string interpolation.

https://dmitripavlutin.com/string-interpolation-in-javascript/#:~:text=String%20interpolation%20is%20replacing%20placeholders,%24%7Bexpression%7D%20as%20a%20placeholder.


e.g.: 
```javascript
header: `<h1 class="text-2xl font-bold">${ this.$i18n.t('emailPolicyTitle') }</h1>`,
```

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]
Don't forget about JIRA tags in commit messages.

**Review by MCFreddie777 — APPROVED** [review]

**Review by Hemii — APPROVED** [review]

**Pull request merged**